packages/strapi-plugin-content-type-builder/controllers/validation/common.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const yup = require('yup');
+
+const VALID_TYPES = [
+  // advanced types
+  'media',
+
+  // scalar types
+  'string',
+  'text',
+  'richtext',
+  'json',
+  'enumeration',
+  'password',
+  'email',
+  'integer',
+  'float',
+  'decimal',
+  'date',
+  'boolean',
```

Comment:
I believe we are also supporting `biginteger`

Reply:
not in the frontend

Reply:
You right! We don't support it in the `content-type-builder` but we do in the `content-manager`...